web/frontend/src/components/slam/index.vue

```diff
@@ -3,16 +3,16 @@
 
 import { $ref, $computed } from 'vue/macros';
 import { grpc } from '@improbable-eng/grpc-web';
-import { toast } from '../lib/toast';
+import { toast } from '@/lib/toast';
```

Comment:
Import aliases are much cleaner that relative path aliases, and result in fewer lines changed when moving around files. We've used them in app for a while and I've been slacking on getting them in RDK. No more!